src/awkward/highlevel.py

```diff
@@ -2224,12 +2333,18 @@ def show(
         valuestr = prettyprint_valuestr(
             self, limit_rows, limit_cols, formatter=formatter_impl
         )
+
+        out = ""
         if type:
             tmp = io.StringIO()
             self.type.show(stream=tmp)
-            out = "type: " + tmp.getvalue() + valuestr
-        else:
-            out = valuestr
+            out += "type: " + tmp.getvalue()
+        if axes:
+            if self.named_axis:
+                out += f"axes: {_prettify_named_axes(self.named_axis)}\n"
+            else:
+                out += "axes: " + ", ".join(map(str, self.positional_axis)) + "\n"
```

Comment:
I'd rather

```suggestion
        if axes and self.named_axis:   # by the way, this is using Truthiness
            out += f"axes: {_prettify_named_axes(self.named_axis)}\n"
```

so if there are no named axes, no space is taken to describe them. (The `self.positional_axis` is just "0, 1, 2, 3, ...", right?)

Also, is `axes=True` in Jupyter?

And another thing, let's make user-facing words consistent: the problem is that it can be hard to guess whether I should say "axis" or "axes". Maybe this argument can be `named_axis: bool`, which would make it more obvious why you wouldn't see any axes when `named_axis=True` and there aren't any _named_ axes. Bottom line: can we refer to these things as "`named_axis`" (singular) everywhere (in all _public_ arguments, methods, properties, ...)?

Reply:
Where do I find the Jupyter setting?